devito/finite_differences/__init__.py

```diff
@@ -1,6 +1,7 @@
-from devito.finite_differences.differentiable import *  # noqa
-from devito.finite_differences.finite_difference import *  # noqa
-from devito.finite_differences.derivative import *  # noqa
-from devito.finite_differences.tools import generate_fd_shortcuts  # noqa
-from devito.finite_differences.coefficients import *  # noqa
-from devito.finite_differences.operators import *  # noqa
+from .differentiable import *  # noqa
+from .finite_difference import *  # noqa
+from .derivative import *  # noqa
+from .tools import generate_fd_shortcuts  # noqa
+from .coefficients import *  # noqa
+from .operators import *  # noqa
+from .equation import *  # noqa
```

Comment:
I am curious. Why the relative import? 

Is it because now each target can implement those classes and with a relative import it can be automatically loaded, something like this?

Reply:
no it's simply "why not", I think relatives are fine in `__init__.py` ?